int32 bReady
int32 bMoving
int32 bError
int32 pos
int32 velocity
int32 ErrorCode
int32 temperature
int32 volt
